definitions/artifacts/elasticsearch-authentication.json

```diff
@@ -30,9 +30,21 @@
     },
     "specs": {
       "type": "object",
+      "required": [
+        "elasticsearch"
+      ],
       "properties": {
         "elasticsearch": {
           "$ref": "../specs/elasticsearch.json"
+        },
+        "aws": {
```

Comment:
should we make this a one of?

Reply:
Eventually we probably should. However, making it a `oneOf` now would effectively make one of them `required`, which would make all our current bundles export an invalid artifact (since almost all of them are missing a cloud spec). I think this is a necessary first step.